api/v1alpha2/types.go

```diff
@@ -431,6 +431,8 @@ type OSDisk struct {
 	OSType      string      `json:"osType"`
 	DiskSizeGB  int32       `json:"diskSizeGB"`
 	ManagedDisk ManagedDisk `json:"managedDisk"`
+	// +optional
+	CachingType string `json:"cachingType"`
```

Comment:
why add this to the v1alpha2 package? All changes to the API should be in the current API version, v1alpha3

Reply:
Good catch, this was a picnic error :sweat_smile: